addons/point_of_sale/views/pos_session_view.xml

```diff
@@ -62,6 +62,13 @@
                             type="object" context="{'balance': 'end'}">
                             <span class="o_stat_text">Set Closing Balance</span>
                         </button>
+                        <button name="action_view_order"
+                            class="oe_stat_button"
+                            attrs="{'invisible': [('order_count', '=', 0)]}"
```

Comment:
I wouldn't make the button invisible in case of 0 orders, we can expect all sessions to always have at least one order anyway (and users might get confused if the field is not always visible)

unless it was a specific request from the PO? didn't see it in the spec